django_rq/management/commands/rqworker.py

```diff
@@ -123,7 +125,10 @@ def handle(self, *args, **options):
             # Close any opened DB connection before any fork
             reset_db_connections()
 
-            w.work(burst=options.get('burst', False), with_scheduler=options.get('with_scheduler', False), logging_level=level)
+            w.work(
+                burst=options.get('burst', False), with_scheduler=options.get('with_scheduler', False),
+                logging_level=level, max_jobs=options['max_jobs']
+            )
```

Comment:
Probably argument values should be retrieved in a consistent way here? Also, since default values are already defined in argument parser, no need to use `options.get()`.

```suggestion
            w.work(burst=options['burst'], with_scheduler=options['with_scheduler'], max_jobs=options['max_jobs'], logging_level=level)
```

Reply:
@ushuz I agree it should be used in a consistent way. Although, `options.get()` is an existing implementation, I have just added `max_jobs` parameter. If you want to follow a particular way, please let me know will change it accordingly.